BPBackendDjango/BPBackendDjango/Views/userviews.py

```diff
@@ -267,6 +287,15 @@ def post(self, request, *args, **kwargs):
                 }
             return Response(data)
 
+        #check if username is allowed
+        if (not all(c in ALLOWED for c in req_data['username'])) or len(req_data['username']) < MIN_USERNAME_LENGTH or str(req_data['username']).startswith(' '):
```

Comment:
Here, a check for the maximum username length is missing

Reply:
no, if username is too long serializer is not valid

Reply:
what do you mean with serializer?